.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,67 @@
+name: Arcaflow CI Workflow
```

Comment:
How does this workflow relate to the one in `build.yml`?

Should that one be invoking this one, once the tests pass (or, should this one be dependent upon the success of that one)?